### DISCUSSION TITLE
directory location of lib and includes installation

### DISCUSSION THREAD

**Discussion by aragon5956** [openssl/openssl#24801] [Q&A]
hello, 
how can i choose the direcotry of lib installation and include directory installation and possibly executable ?

if i type `./Configure --help` i've this : `alexandre@alexandre-developpeur:~/Téléchargements/openssl$ ./Configure --help
Configuring OpenSSL version 3.4.0-dev for target linux-x86_64
Using os-specific seed configuration
Created configdata.pm
Running configdata.pm
Created Makefile.in
Created Makefile

**********************************************************************
***                                                                ***
***   OpenSSL has been successfully configured                     ***
***                                                                ***
***   If you encounter a problem while building, please open an    ***
***   issue on GitHub <https://github.com/openssl/openssl/issues>  ***
***   and include the output from the following command:           ***
***                                                                ***
***       perl configdata.pm --dump                                ***
***                                                                ***
***   (If you are new to OpenSSL, you might want to consult the    ***
***   'Troubleshooting' section in the INSTALL.md file first)      ***
***                                                                ***
**********************************************************************
alexandre@alexandre-developpeur:~/Téléchargements/openssl$ perl configdata.pm
Created Makefile.in
Created Makefile
alexandre@alexandre-developpeur:~/Téléchargements/openssl$ 
`, but i can't choose , and i don't know how ?
Could you help me ?
Regards

**Comment by cchinchole** [comment]
./Configure --openssldir=/home/username/src/openssl/build/ --prefix=/home/username/src/openssl/build/ -Wl,-rpath=/home/username/src/openssl/build/lib64/
make test
make install

This is how I build OpenSSL. I left the path to show you need to use absolute paths when configuring.

**Comment by aragon5956** [comment]
ok, i found it !! the command is : 

`./Configure build_docs
`
regards
